src/Codeception/Command/Run.php

```diff
@@ -61,6 +61,7 @@
  *  --html                Generate html with results (default: "report.html")
  *  --xml                 Generate JUnit XML Log (default: "report.xml")
  *  --phpunit-xml         Generate PhpUnit XML Log (default: "phpunit-report.xml")
+ *  --prefer-local        Prefer a local to a Composer-installed version
```

Comment:
This description could be better, what is that `local`?

Reply:
```suggestion
 *  --prefer-local        Insist on using a local version over the one defined in composer.json
```

Reply:
I think that I disagree with usage of `local` altogether, your `local` can be a global composer installation or phar file somewhere in PATH.

Reply:
> This description could be better, what is that `local`?

A local version to a Composer-installed version.

Reply:
`prefer-local` was suggested by @DavertMik

Reply:
@Naktibalda how would you call it?

Reply:
From my point of view, codeception in `vendor` is as local as possible, the orther thing is less local.
`--no-redirect` is technical, but it is straitght to the point.
A suggestion to use `--no-redirect` option could be added to `Redirecting to Composer-installed version in vendor/codeception` message.

Reply:
Redirect can be confused with browser redirection. 
So I'd like to have something better here. 

`--prefer-current`
`--no-vendor`